docs/access-management/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Role Based Access Control (RBAC)
+
+Role Based Access Control allows a user with Administrative access to apply roles to users, granting different levels of permission within an organization or workspace. 
+
+## Workspace Resource Roles
+A user can have at most one role of this kind per workspace. Permissions are scoped to the specific workspace in which the user has this role.
+
+| Permissions           | WorkspaceReader    | WorkspaceEditor    | WorkspaceAdmin    |
+| ---------------------- | :--------: | :--------:| :--------: |
+| **ReadWorkspace**<br />-List the connections in a workspace<br />-Read individual connections<br />-Read workspace settings (data residency, users, connector versions, notification settings) | X | X | X |
+| **Update Connection**<br />-Start/cancel syncs<br />- Modify a connection, including name, replication settings, normalization, DBT<br />- Delete a connection<br />- Create/Update/Delete connector builder connectors |  | X | X |
```

Comment:
Is it worth noting that these users can also create and edit sources/destinations?

Reply:
So, that applies to `WorkspaceReader`, `WorkspaceEditor`, and `WorkspaceAdmin`?, but is not a defined permission? If so, I think that makes sense to put in the text above the table.

Reply:
No, sorry!  

"Modify connector settings" (col J in the RBAC spreadsheet) is the ability to create, modify, and delete sources and destinations in a workspace.  

It is available to Workspace Editor and Workspace Admin.